SpatialGDK/Source/SpatialGDKEditor/Public/SpatialGDKEditorSettings.h

```diff
@@ -339,6 +339,10 @@ class SPATIALGDKEDITOR_API USpatialGDKEditorSettings : public UObject
 	UPROPERTY(EditAnywhere, config, Category = "Launch", meta = (DisplayName = "Auto-start local deployment"))
 	bool bAutoStartLocalDeployment;
 
+	/** Stop play in editor when Automation Manager finishes running Tests. If false, the native Unreal Engine behaviour maintains of leaving the last map PIE running. */
+	UPROPERTY(EditAnywhere, config, Category = "Launch", meta = (DisplayName = "Stop play in editor on Testing completed"))
+	bool bStopPIEOnTestingCompleted;
```

Comment:
Set the default to true?

Reply:
IMO it should be yes, just kept it false as it maintains the UE behaviour. @mironec what do you think, default to on?

Reply:
FYI changed the default to true, if afterwards we want to change just let me know.